content/docs/introduction/faq.md

```diff
@@ -0,0 +1,103 @@
+---
+title: FAQ
+sort_rank: 5
+---
+
+# Frequently Asked Questions
+
+## General
+
+### What is Prometheus?
+Prometheus is an open-source systems monitoring and alerting toolkit,
+with an active ecosystem. See the [overview](../overview).
+
+### How does Prometheus compare against \[other monitoring system\]?
+
+See the [comparison](../comparison) page.
+
+### What dependencies does Prometheus have?
+
+The main Prometheus server runs standalone and has no external dependencies.
+
+### Can Prometheus be made highly available?
+
+Yes, run identical Prometheus servers on two or more separate machines.
+Identical alerts will be deduplicated by the [Alertmanager](https://github.com/prometheus/alertmanager).
+
+The Alertmanager cannot currently be made highly available, but this is a goal.
+
+### What language is Prometheus written in?
+
+Most Prometheus components are written in Go. Some are also written in Java and Ruby.
+
+### Who wrote Prometheus?
+
+Prometheus was initially started privately by
+[Matt Proud](https://github.com/matttproud) and
+[Julius Volz](https://github.com/juliusv). The majority of its
+development has been sponsored by [SoundCloud](https://soundcloud.com).
+
+Other companies making active contributions include [Boxever](www.boxever.com)
+and [Docker](https://www.docker.com). A full list can be found in the
+[CONTRIBUTORS](https://github.com/prometheus/prometheus/blob/master/CONTRIBUTORS.md)
```

Comment:
Good question. So far, our own Prometheus OSSing strategy doc says, "A separate AUTHORS file will list all past/present contributors, and will also clarify how the project got started by Matt T. Proud. This would also be the right place to mention SoundCloud’s sponsorship (decision needed)". Although I can't find anywhere that an AUTHORS file is mandatory with the Apache license, legal had recommended the following in applying the license:

"If SoundCloud has to be mentioned at all, I agree to your suggestion using "The Prometheus Authors" and having a separate authors file that lists them all and links to SoundCloud Ltd. only as sponsor or supporter. This especially makes sense since both SC employees and externals will contribute."

In light of both of these points, I think it's probably worth keeping it (but rename to AUTHORS, as that is more common) around. As for differentiating contributors and committers, maybe just every committer is a contributor, but we could make a rule. Look at Docker's AUTHORS file for example: https://github.com/docker/docker/blob/master/AUTHORS

/cc @beorn7